File: poo/2. User Registration at a Gym/essential.php
<?php
    class Essential extends CalculateTotal {
        private int $hour;
        private int $value;
        private int $calculate;

        public function __construct(int $hour) {
            parent::__construct("Essential", "Basico");
            $this->hour = $hour;
            $this->value = 30000;
        }

        public function getHour(): int {
            return $this->hour;
        }
        public function getValue(): int {
            return $this->value;
        }

        public function calculate(): float {
            $this->calculate = $this->hour * $this->value;
            return $this->calculate;
        }
    }
?>

File: poo/2. User Registration at a Gym JS/essential.java
public class Essential extends CalculateTotal {
    private int hour;
    private int value;
    private int calculate;

    public Essential(int hour) {
        super("Essential", "Basico");
        this.hour = hour;
        this.value = 30000;
    }

    public int getHour() {
        return hour;
    }

    public int getValue() {
        return value;
    }

    @Override
    public double calculate() {
        calculate = hour * value;
        return calculate;
    }
}
